railties/test/application/test_runner_test.rb

```diff
@@ -728,17 +728,17 @@ def test_run_in_parallel_with_processes
         end
       RUBY
 
-      output = run_test_command(file_name)
+      output = run_test_command(file_name, allow_failure: false)
 
-      assert_match(/Finished in.*2 runs, 2 assertions/m, output)
+      assert_match(/Finished in.*2 runs, 2 assertions, 0 failures, 0 errors, 0 skips/m, output)
       assert_match %r{Running \d+ tests in parallel using \d+ processes}, output
       assert_no_match "create_table(:users)", output
     end
 
     def test_parallelization_is_disabled_when_number_of_tests_is_below_threshold
       exercise_parallelization_regardless_of_machine_core_count(with: :processes, threshold: 100)
 
-      file_name = create_parallel_processes_test_file
+      file_name = create_parallel_blank_test_file
```

Comment:
The test generated by this test fails currently (it's just ignored).

If this PR isn't accepted, I can extract these test fixes and the typo fix above into a separate PR.